autonomy/cli/analyse.py

```diff
@@ -165,13 +160,13 @@ def docstrings(ctx: Context, update: bool) -> None:
             raise click.ClickException(str(e)) from e
 
 
-@abci_group.command(name="logs")
+@analyse_group.command(name="logs")
```

Comment:
Since we've moved all other subcommands out of the `abci` there is no point in just keeping one last command.